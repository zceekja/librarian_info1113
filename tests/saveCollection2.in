add collection book1.csv
save collection output.csv