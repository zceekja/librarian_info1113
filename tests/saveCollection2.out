user: 17 books successfully added.

user: Success.

user: 